provision/phpcs/composer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "VVV/phpcs",
+    "type": "project",
+    "require": {
+        "oomphinc/composer-installers-extender": "^1.1",
+        "squizlabs/php_codesniffer": "2.*",
```

Comment:
I think requiring just `wp-coding-standards/wpcs` is enough, as `wpcs` requires `squizlabs/php_codesniffer`